micrometer-core/src/test/java/io/micrometer/core/instrument/HighCardinalityTagsDetectorTests.java

```diff
@@ -107,4 +122,21 @@ public String getName() {
 
     }
 
+    private static class TestCustomMeterNameConsumer implements Consumer<String> {
+
+        @Nullable
```

Comment:
Why do you need such annotations in test code? Seems like an overkill to me.

Reply:
It's a good practice even in tests, also, it will eliminate IDE complaints/warnings.